README.md

```diff
@@ -1,52 +1,48 @@
-# Google Developer Student Club - LBCE Website
+<h1 align="center"> Google Developer Student Club - LBCE Website</h1>
 
 ![GitHub issues](https://img.shields.io/github/issues/Gdsc-Lbce/GDSC_LBCE_Website)
 ![GitHub pull requests](https://img.shields.io/github/issues-pr/Gdsc-Lbce/GDSC_LBCE_Website)
 [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](http://makeapullrequest.com)
 <br />
 <br />
-<div align="center" margin-bottom="10em"><a href="https://hacktoberfest.digitalocean.com/"><img src="https://hacktoberfest.digitalocean.com/_nuxt/img/logo-hacktoberfest-full.f42e3b1.svg" alt="Banner" width="60%"/></a></div>
-<br />
-<br />
-The official website of GDSC LBCE community.
+<div align="center" margin-bottom="10em"><a href="https://gdsc.community.dev/dr-lankapalli-bullayya-college-of-engineering-visakhapatnam/"><img src="https://user-images.githubusercontent.com/75005433/145153403-4472e4df-88cf-41c5-8631-97ab86a40267.png" alt="Banner" width="60%"/></a></div>
+
+
+<h2 align="center">About the Repository</h2>
+The Repository belongs to Google Developer Students Club of Dr. Lankapalli Bullayya College of Engineering.
+This is a newly found GDSC club and we are looking for some contributors to help us build our community website from scratch. We always encourage new ideas. Feel free to get in touch with us and join our [discord server](https://discord.gg/ATvz2fqw) for updates.
```

Comment:
@Vedant-Jayesh-Oza, u need to edit the discord server link it is not showing properly.